.vscode/settings.json

```diff
@@ -14,4 +14,5 @@
   "editor.wordWrap": "on",
   "typescript.updateImportsOnFileMove.enabled": "always",
   "files.eol": "\n",
+  "cSpell.words": ["Parens"]
```

Comment:
parens?

Reply:
Parenthesis, refers to allowParens rule in .prettierrc

Reply:
ok